<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatorWindow</class>
 <widget class="QMainWindow" name="CalculatorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background-color: #F4F4F4;</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>弥散方程求波长</string>
     </attribute>
     <widget class="QGroupBox" name="DispersionEquationCalculator">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>370</width>
        <height>206</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid rgb(100,100,100);
border-radius:15px;
background-color: #F4F4F4;</string>
      </property>
      <property name="title">
       <string>弥散方程迭代计算求波长</string>
      </property>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>76</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px solid rgb(100,100,100);
border-radius:0px;
background-color: #F4F4F4;</string>
       </property>
       <property name="text">
        <string>迭代次数上限</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>76</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px solid rgb(100,100,100);
border-radius:0px;
background-color: #F4F4F4;</string>
       </property>
       <property name="text">
        <string>波周期</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>76</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px solid rgb(100,100,100);
border-radius:0px;
background-color: #F4F4F4;</string>
       </property>
       <property name="text">
        <string>水深</string>
       </property>
      </widget>
      <widget class="QPushButton" name="DE_Calc">
       <property name="geometry">
        <rect>
         <x>275</x>
         <y>20</y>
         <width>80</width>
         <height>175</height>
        </rect>
       </property>
       <property name="text">
        <string>计算</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>76</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px solid rgb(100,100,100);
border-radius:0px;
background-color: #F4F4F4;</string>
       </property>
       <property name="text">
        <string>初始波长</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>76</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px solid rgb(100,100,100);
border-radius:0px;
background-color: #F4F4F4;</string>
       </property>
       <property name="text">
        <string>目标迭代精度</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>76</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px solid rgb(100,100,100);
border-radius:0px;
background-color: #F4F4F4;</string>
       </property>
       <property name="text">
        <string>输出</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="DE_Cycle">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid rgb(100,100,100);
border-radius:3px;
background-color: #F4F4F4;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="DE_Depth">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid rgb(100,100,100);
border-radius:3px;
background-color: #F4F4F4;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="DE_InitialWaveLength">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid rgb(100,100,100);
border-radius:3px;
background-color: #F4F4F4;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="DE_Output">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid rgb(100,100,100);
border-radius:3px;
background-color: #F4F4F4;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="DE_IterationAccuracy">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>140</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid rgb(100,100,100);
border-radius:3px;
background-color: #F4F4F4;</string>
       </property>
       <property name="text">
        <string>0.001</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="DE_MaximumIterations">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>170</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid rgb(100,100,100);
border-radius:3px;
background-color: #F4F4F4;</string>
       </property>
       <property name="text">
        <string>1000</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>椭圆余弦波波长计算</string>
     </attribute>
     <widget class="QGroupBox" name="EllipticCosineWaveErgodicCalculator">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>370</width>
        <height>206</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid rgb(100,100,100);
border-radius:15px;
background-color: #F4F4F4;</string>
      </property>
      <property name="title">
       <string>椭圆余弦波波长遍历计算</string>
      </property>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>76</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px solid rgb(100,100,100);
border-radius:0px;
background-color: #F4F4F4;</string>
       </property>
       <property name="text">
        <string>κ输出</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>76</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px solid rgb(100,100,100);
border-radius:0px;
background-color: #F4F4F4;</string>
       </property>
       <property name="text">
        <string>波周期</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>76</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px solid rgb(100,100,100);
border-radius:0px;
background-color: #F4F4F4;</string>
       </property>
       <property name="text">
        <string>水深</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ECWE_Calc">
       <property name="geometry">
        <rect>
         <x>275</x>
         <y>20</y>
         <width>80</width>
         <height>175</height>
        </rect>
       </property>
       <property name="text">
        <string>计算</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>76</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px solid rgb(100,100,100);
border-radius:0px;
background-color: #F4F4F4;</string>
       </property>
       <property name="text">
        <string>波高</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>76</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px solid rgb(100,100,100);
border-radius:0px;
background-color: #F4F4F4;</string>
       </property>
       <property name="text">
        <string>遍历精度</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>76</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px solid rgb(100,100,100);
border-radius:0px;
background-color: #F4F4F4;</string>
       </property>
       <property name="text">
        <string>波长输出</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ECWE_WavePeriod">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid rgb(100,100,100);
border-radius:3px;
background-color: #F4F4F4;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ECWE_Depth">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid rgb(100,100,100);
border-radius:3px;
background-color: #F4F4F4;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ECWE_WaveHeight">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid rgb(100,100,100);
border-radius:3px;
background-color: #F4F4F4;</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="ECWE_WaveLengthOutput">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid rgb(100,100,100);
border-radius:3px;
background-color: #F4F4F4;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="ECWE_ErgodicAccuracy">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>170</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid rgb(100,100,100);
border-radius:3px;
background-color: #F4F4F4;</string>
       </property>
       <property name="text">
        <string>0.001</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ECWE_KappaOutput">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>140</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid rgb(100,100,100);
border-radius:3px;
background-color: #F4F4F4;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>圆锥底岸波浪折射计算</string>
     </attribute>
     <widget class="QGroupBox" name="EllipticCosineWaveErgodicCalculator_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>370</width>
        <height>206</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid rgb(100,100,100);
border-radius:15px;
background-color: #F4F4F4;</string>
      </property>
      <property name="title">
       <string>椭圆余弦波波长遍历计算</string>
      </property>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>76</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px solid rgb(100,100,100);
border-radius:0px;
background-color: #F4F4F4;</string>
       </property>
       <property name="text">
        <string>κ输出</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>76</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px solid rgb(100,100,100);
border-radius:0px;
background-color: #F4F4F4;</string>
       </property>
       <property name="text">
        <string>波周期</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>76</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px solid rgb(100,100,100);
border-radius:0px;
background-color: #F4F4F4;</string>
       </property>
       <property name="text">
        <string>水深</string>
       </property>
      </widget>
      <widget class="QPushButton" name="WR_Calc">
       <property name="geometry">
        <rect>
         <x>275</x>
         <y>20</y>
         <width>80</width>
         <height>175</height>
        </rect>
       </property>
       <property name="text">
        <string>计算</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_28">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>76</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px solid rgb(100,100,100);
border-radius:0px;
background-color: #F4F4F4;</string>
       </property>
       <property name="text">
        <string>波高</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_29">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>76</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px solid rgb(100,100,100);
border-radius:0px;
background-color: #F4F4F4;</string>
       </property>
       <property name="text">
        <string>遍历精度</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_30">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>76</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px solid rgb(100,100,100);
border-radius:0px;
background-color: #F4F4F4;</string>
       </property>
       <property name="text">
        <string>波长输出</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ECWE_WavePeriod_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid rgb(100,100,100);
border-radius:3px;
background-color: #F4F4F4;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ECWE_Depth_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid rgb(100,100,100);
border-radius:3px;
background-color: #F4F4F4;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ECWE_WaveHeight_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid rgb(100,100,100);
border-radius:3px;
background-color: #F4F4F4;</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="ECWE_WaveLengthOutput_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid rgb(100,100,100);
border-radius:3px;
background-color: #F4F4F4;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="ECWE_ErgodicAccuracy_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>170</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid rgb(100,100,100);
border-radius:3px;
background-color: #F4F4F4;</string>
       </property>
       <property name="text">
        <string>0.001</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ECWE_KappaOutput_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>140</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid rgb(100,100,100);
border-radius:3px;
background-color: #F4F4F4;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QChartView" name="WaveView">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>661</width>
      <height>465</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid rgb(100,100,100);
border-radius:15px;
background-color: #F4F4F4;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>491</y>
      <width>1051</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid rgb(100,100,100);
border-radius:15px;
background-color: #F4F4F4;</string>
    </property>
    <property name="title">
     <string>日志区</string>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>1031</width>
       <height>116</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px solid rgb(100,100,100);
border-radius:15px;
background-color: #FEFEFE;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
